surveys/static/js/survey_analysis.js

```diff
@@ -0,0 +1,209 @@
+$(document).ready(function () {
```

Comment:
I like the idea of transferring this logic to its own file, but it looks like this code is duplicated in the `survey_submitted_detail.html` template. Did you mean to keep them both?